video/out/cocoa/events_view.m

```diff
@@ -96,6 +99,25 @@ - (BOOL)containsMouseLocation
     return CGRectContainsPoint(clippedBounds, pt);
 }
 
+- (void)updateCursorForceVisible:(BOOL)forceVisible
+{
+    if ((forceVisible || self.wantCursorVisible) && !self.cursorVisible) {
+        [NSCursor unhide];
+        self.cursorVisible = YES;
+    } else if (!self.wantCursorVisible && [self canHideCursor] &&
+                self.cursorVisible) {
```

Comment:
Minor style nit: the { should be on the next line.

Otherwise the pr is fine with me I guess.